src/api/java/appeng/api/util/IClientHelper.java

```diff
@@ -0,0 +1,21 @@
+
+package appeng.api.util;
+
+
+import java.util.List;
+
+import appeng.api.storage.ICellInventoryHandler;
+import appeng.api.storage.data.IAEStack;
+
+
+public interface IClientHelper
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Annotate the "IClientHelper" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1609)